mvcc/backend/metrics.go

```diff
@@ -83,6 +83,17 @@ var (
 		// highest bucket start of 0.01 sec * 2^16 == 655.36 sec
 		Buckets: prometheus.ExponentialBuckets(.01, 2, 17),
 	})
+
+	batchIntervalSec = prometheus.NewHistogram(prometheus.HistogramOpts{
+		Namespace: "etcd_debugging",
+		Subsystem: "backend",
+		Name:      "batch_interval_duration_milliseconds",
```

Comment:
variable name suggests seconds but metric name suggests milliseconds.  For prometheus it's okay to use seconds here. There's a convention: https://prometheus.io/docs/practices/naming/#base-units